docs/using-draft-js.stories.mdx

```diff
@@ -0,0 +1,121 @@
+`TextEditor` component's interface.
+
+```js
+import { EditorState } from 'draft-js';
+```
+```js
+import { TextEditorState } from 'carbon-react/lib/components/text-editor';
+```
+
+#### Useful static methods
+The framework surfaces a range of static methods for initialising the state: 
+* `EditorState.createEmpty(decorator?: DraftDecoratorType)` - will intitialise the component with a new `EditorState` 
+object with an empty `ContentState` and any `Decorators` passed to it.
+* `EditorState.createWithContent(contentState: ContentState, decorator?: DraftDecoratorType)` - is used to initialise 
+the component with some existing `ContentState` and any `Decorators` and return a new `EditorState` object.
+* `EditorState.create(config: EditorStateCreationConfig)` - offers the same as `createWithContent` but enables 
+initialising the `EditorState` from a config, affording you more fine grain control. For example, it is possible to 
+define an intitial `SelectionState` using this static method.
+
+#### Other useful methods
+* `EditorState.push(editorState: EditorState, contentState: ContentState, changeType: EditorChangeType)` - it is very
+unlikely that you will have a need to use this but if direct content changes are required they must be applied to the 
+`EditorState` using this method. It returns a new `EditorState` object with the specified `ContentState` applied as the 
+new currentContent, the `changeType` defines the operation being carried out on the state, see 
+https://draftjs.org/docs/api-reference-editor-change-type for the defined options.
+* The current `ContentState` can be accessed by calling the `getCurrentContent()` instance method.
+* The current `SelectionState` can be accessed by calling the `getSelection()` instance method.
+
+### ContentState
+`ContentState` is also an Immutable Record and represents the state of the editor's entire contents (text, block and 
+inline styles, and entity ranges) and its two selection states (before and after rendering). There are range of useful 
+methods accessible through `ContentState` that can provide information about the current content rendered in the editor, 
+for more dtsails it is best to refer to the API reference documentation 
+https://draftjs.org/docs/api-reference-content-state.
+
+#### Importing ContentState
+The `ContentState` can be imported either directly from `draft-js` or alternatively it has been exposed as part of the 
+`TextEditor` component's interface.
+
+```js
+import { ContentState } from 'draft-js';
+```
+```js
+import { TextEditorContentState } from 'carbon-react/lib/components/text-editor';
+```
+
+#### Useful static methods
+There are two statci methods that facilitate creating `ContentState`:
+* `createFromText(text: string, delimiter?: string)` - will generate state from a given string paramater, passing an 
+optional delimiter will define how the content blocks are split, if none isd provided the method will default to using 
```

Comment:
Typo on "is" although I think it should be "are" anyway - although grammar isn't my strong point!

Reply:
I'm referring to the delimiter, I'll try and tweak it better as it does read a little confusingly